### PR TITLE
Add Alumni Section on Landing Page

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Carousel with alumni's information has been added
image

Latest version:
image

### CODE REVIEW

**Comment by Kevin-Tan25** [comment]
Overall looks really good, could you add more padding between Name and Title?
Also the field of study looks a bit too close near the edge of the container, was wondering if you could either make text smaller, shift the text up (more horizontal space), or just increase the width of the container?
Screenshot 2023-07-18 at 1 16 26 PM
8594/e2ec4aaf-f74d-4226-8864-6d81ed83cdd3">

File: src/components/Alumni/Alumni.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+function AlumniHeader() {
+    return (
+        <div  className="flex flex-col items-center gap-y-4">
+            <h1 className="text-h1 font-header">Careers - where are our alumni?</h1>
+            <h3 className="text-h3">Our programs prepare you for a career as product manager! </h3>
+        </div>
+    )
+}
+
+function SeeAllFeature({className}) {
```
Comment (Kevin-Tan25) [inline src/components/Alumni/Alumni.jsx:23]: I can take this out and make this into a shared component in my next MR

File: src/components/Alumni/AlumniCard.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        currentTarget.src = EmptyProfile
+    }
+    const logoLib = {
+        "Microsoft": Microsoft,
+    }
+    return (
+        <div className="w-[23.76669rem] h-[13.03331rem] rounded-[6.25rem] bg-[#2B3950]
+            flex flex-row py-[2.06rem] content-center
+            font-content">
+            <img src={img} onError={imgOnError} className="w-[8rem] h-[8rem] rounded-full mt-[0.56rem] ml-[1.69rem] mr-[1.8rem]
```
Comment (Kevin-Tan25) [inline src/components/Alumni/AlumniCard.jsx:18]: just curious... what's the onError usecase for? is it for e.g. corrupted images that can't load?

File: src/components/Alumni/Alumni.jsx (hunk 1 above)
Comment (IcedTea2K) [reply src/components/Alumni/Alumni.jsx:23]: Sounds good. Should I leave it as is rn?

File: src/components/Alumni/AlumniCard.jsx (hunk 2 above)
Comment (IcedTea2K) [reply src/components/Alumni/AlumniCard.jsx:18]: Yeap, you're right. Or more specifically, somehow the path is corrupted, and the img can't be found.

**Comment by IcedTea2K** [comment]
@Kevin-Tan25
Before:
image
After:
image

File: src/components/Alumni/AlumniCard.jsx (hunk 2 above)
Comment (Kevin-Tan25) [reply src/components/Alumni/AlumniCard.jsx:18]: Makes sense, good addition to make code robust thanks for explaining

File: src/components/Alumni/Alumni.jsx (hunk 1 above)
Comment (Kevin-Tan25) [reply src/components/Alumni/Alumni.jsx:23]: Yep let's leave it

**Review by Kevin-Tan25 — APPROVED** [review]
Looks good!

**Pull request merged**